package.json

```diff
@@ -11,7 +11,7 @@
     "build": "npm run extract-roles && babel -d dist src --copy-files",
     "prepare": "npm run build",
     "test": "ava",
-    "lint": "eslint src test/commands && documentation lint src test/commands",
+    "lint": "eslint src test/commands",
```

Comment:
We may still want to leave `documentation lint src` :)

Reply:
Thank you Brett!